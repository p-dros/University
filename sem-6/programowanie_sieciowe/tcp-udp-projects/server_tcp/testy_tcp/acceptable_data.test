kilKa SloW paLinDroMow cZy teZ nie Sys radAr dzwoN czy HAh
KoleJne ZapyTanie zaWierajaCE JakIes dANe
KajAk WodA ALa maJonEZ roSzpUNkA
