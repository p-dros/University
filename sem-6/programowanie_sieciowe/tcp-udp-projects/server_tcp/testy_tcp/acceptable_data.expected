3/11
0/5
2/5
